docs/content/en/docs/concepts/tracing-policy/example.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Example"
+icon: "tutorials"
+weight: 1
+description: "Learn the basics of Tracing Policy via an example"
+---
+
+To discover `TracingPolicy`, let's understand via an example that will be
+explained, part by part, in this document:
+
+```yaml
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "fd-install"
+spec:
+  kprobes:
+  - call: "fd_install"
+    syscall: false
+    args:
+    - index: 0
+      type: "int"
+    - index: 1
+      type: "file"
+    selectors:
+    - matchArgs:
+      - index: 1
+        operator: "Equal"
+        values:
+        - "/etc/passwd"
+        - "/etc/shadow"
```

Comment:
maybe it's better if we don't use enforcement on the example? I don't know on one side I wanted something simple but on the other side I wanted to showoff Tetragon's capabilities. Maybe removing sigkill is better.

Reply:
I think using sigkill is good for illustration purposes: i) it illustrates how one can use actions, and ii) you can just see the application being killed rather than an event which is more immediate.

Reply:
Yep I used a dummy `/tmp/tetragon` file instead